bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestBotFactory.java

```diff
@@ -116,7 +116,7 @@ public List<Bot> create(BotConfiguration configuration) {
             var censusRef = configuration.repositoryRef(repo.value().get("census").asString());
             var repository = configuration.repository(repo.name());
             var botBuilder = PullRequestBot.newBuilder()
-                                           .repo(repository)
+                                           .repo(configuration.repository(repo.name()))
```

Comment:
Why this change? The variable repository is defined on the line above.

Reply:
When merging SKARA-1850 to this one, I must made a mistake. Thanks for catching it!